### ISSUE TITLE
gnome-initial-setup should depend on cracklib

### ISSUE THREAD

**Issue by alexjp** [voidlinux/void-packages#3328]
Hi,

Currently installed voidlinux in a virtual machine using the basic iso ( no desktop environment iso ).

Upon installing gnome, i was greeted by the gnome-initial-setup. In the passwords screen, it never accepts the password as a good password, because cracklib cannot be found, and not updating the password strength meter.

Could cracklib be included in the gnome-initial-setup package as a dependency ?

Thanks !

**Comment by xtraeme** [comment]
Did you verify that just installing `cracklib` was enough to get this working?

**Comment by alexjp** [comment]
Yes. Upon manually installing cracklib, it worked ( gnome-initial-setup detected properly my password strength and allowed me to continue ).
